### ISSUE TITLE
no-hashtag-description does not allow for eslint directives

### ISSUE THREAD

**Issue by ilyavolodin** [dotansimha/graphql-eslint#317]
I can't seem to get `no-hashtag-description` rule to allow any hashtag comments at all, including eslint directives. Here are two examples that are erroring out for me:
```graphql
# eslint-disable @graphql-eslint/no-unreachable-types

"Types of 'unlessContainsTypes' omit"
enum OmitTypes {
  "Scalar fields"
  scalar
  "Complex type fields"
  nonScalar
}
```

```graphql
# eslint-disable-next-line @graphql-eslint/description-style
"""
The different roles a funding instrument can have, depending on different features that Venmo offers.
Usually it's either used as the default instrument, as a backup if the default cannot be used, or in some features,
it indicates if the funding instrument can be used for that feature
"""
type FundingInstrumentRoles {
...
}
```

I would suggest to modify the rule to exclude any hashtag comments that contain eslint directives, i.e. `# eslint` should always be excluded from this rule.

**Comment by dotansimha** [comment]
Sure, totally makes sense :) Thanks @ilyavolodin

**Comment by dotansimha** [comment]
Fixed in `@graphql-eslint/eslint-plugin@0.8.1`
